docs/pbc/all/business-intelligence/202410.0/amazon-quicksight-third-party-integration/back-office-refresh-analytics-datasets.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Back Office: Refresh analytics datasets"
+description: Learn how to refresh analytics datasets in the Back Office.
+last_updated: Oct 24, 2024
+template: back-office-user-guide-template
+related:
+---
+
+This document describes how to refresh datasets manually and on a schedule.
```

Comment:
let's please add here a small section talking about SPICE (https://docs.aws.amazon.com/quicksight/latest/user/spice.html)

I think users need to understand what it is _in general_, as it is a paid line item (500 GB is included, we don't have to mention it) in the QS SKU. 

the main idea is to talk about the benefits of SPICE:
(from AWS page):
- Your analytical queries process faster.
- You don't need to wait for a direct query to process.
- Data stored in SPICE can be reused multiple times without incurring additional costs. If you use a data source that charges per query, you're charged for querying the data when you first create the dataset and later when you refresh the dataset.



![Screenshot 2024-11-15 at 16 48 58](https://github.com/user-attachments/assets/63d7c476-8078-4dbf-a163-9c882787d2bc)

Reply:
we also need to add the possibility on how to check the current SPICE usage in the top right corner under QS > Datasets > New Dataset.

Reply:
Are you referring to this?
<img width="451" alt="image" src="https://github.com/user-attachments/assets/3ffa6d48-0107-414c-9637-a1f7be1380fc">

Reply:
Yes, I believe it is the only way how to check the capacity for the customer.